### PR TITLE
Updated CONTRIBUTING.md with Code Formatting Guidelines

### PR DESCRIPTION
**Pull request by suddendust**
## Description
This PR updates the contributions guide with guidelines around code formatting.
## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -71,6 +71,11 @@ Please ensure your code is adequately documented. Some things to consider for do
 * Use logging levels correctly: set level to `debug` for verbose logs useful for only for debugging.
 * Do not log stack traces via `printStackTrace` method of the exception.
 
+#### Code Formatting
+* Ensure that the code you add is properly formatted as per the Pinot style-sheets.
+* Pinot's style-sheets can be found in ```incubator-pinot/config```.
+* For information on how to import the style-sheet into your IDE, refer [here](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#intellij).
```
Comment (jackjlli) [inline CONTRIBUTING.md:77]:
```suggestion
* For information on how to import the style-sheet into your IDE, refer [here](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup#setup-ide).
```

**Review by jackjlli — APPROVED** [review]
Minor but LGTM

**Comment by suddendust** [comment]
> Minor but LGTM

Have committed the suggested change, thanks :). How can these changes be merged?

**Comment by suddendust** [comment]
Hey hi @jackjlli can this be merged please if there are no further changes to make? Thanks :)

**Review by jackjlli — APPROVED** [review]
LGTM. Thanks for updating the doc!

**Pull request merged**